client/src/components/Results/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import resultsImg from '../../assets/results.png'
+import resutlsData from '../resultsData'
+import Button from '../sharedComponent/Button'
+import {
+  Container,
+  Header,
+  TypeContainer,
+  Img,
+  Type,
+  Description
+} from './style.Results'
+
+class Results extends Component {
+  render() {
+    const { id } = this.props.match.params
+    return (
+      <Container>
+        <Header>Results</Header>
+        <Img src={resultsImg} atl="results" />
+        {resutlsData.map(result =>
```

Comment:
we should always check for existence of an array before running a map function on it to avoid throwing an error.
```suggestion
        {resultsData && resutlsData.map(result =>
```